magento/magento2
# #4154 - There is no ability to add a tab to product page at a desired position

### Description (*)
Solves issue when you cannot sort product information tabs on product detail page.
Added the ability to set group block argument `sort_order` And fetch result based on that.

### Fixed Issues (if relevant)
1. magento/magento2#4154 - There is no ability to add a tab to product page at a desired position


### Manual testing scenarios (*)
Preconditions: product with description and some attributes with values that are visible on product page.
Steps:
1. Open `app/code/Magento/Catalog/view/frontend/layout/catalog_product_view.xml`
2. Set to block with name `product.info.description` attribute `after="product.attributes"`
3. Flush cache
4. Open product detail page
5. Tab "More Information" should be displayed instead of "Details".

To solve the issue and don't change `.phtml` output a lot, was introduced block argument `sort_order` and do the sorting based on it. Default sort order is set to 0.

### Contribution checklist (*)
